share/instruments/instruments.xml

```diff
@@ -5143,8 +5143,8 @@
                   <genre>marching</genre>
             </Instrument>
             <Instrument id="snare-drum">
-                  <longName>Snare Drum</longName>
-                  <shortName>Sn. Dr.</shortName>
+                  <longName>Concert Snare Drum</longName>
```

Comment:
conflicts with mtest for importmidi, see https://travis-ci.org/musescore/MuseScore/jobs/363144548#L5031-L5065